tests/sentry/middleware/integrations/parsers/test_jira_server.py

```diff
@@ -84,6 +85,70 @@ def test_routing_endpoint_with_integration(self):
             region_names=[region.name],
         )
 
+    @override_regions(region_config)
+    @override_settings(SILO_MODE=SiloMode.CONTROL)
+    @override_options({"hybridcloud.webhookpayload.use_mailbox_buckets": True})
+    @responses.activate
+    def test_routing_webhook_with_mailbox_buckets_low_volume(self):
+        route = reverse("sentry-extensions-jiraserver-issue-updated", kwargs={"token": "TOKEN"})
+
+        request = self.factory.post(
+            route, data=issue_updated_payload, content_type="application/json"
+        )
+        parser = JiraServerRequestParser(request=request, response_handler=self.get_response)
+
+        OrganizationMapping.objects.get(organization_id=self.organization.id).update(
+            region_name="us"
+        )
```

Comment:
Could we just create a new organization in the US region here instead of updating the mapping directly? The factory should have a convenience prop to se this on the org. My concern is we end up breaking down the line if we rely on other related org data in the future.

Reply:
Yes, this is clunky right now because the organization is generated during setup() when we don't have the region list configured. I can move the region configuration up though.